site/content/includes/quickstart-prometheus-steps.md

```diff
@@ -0,0 +1,63 @@
+This quickstart uses [the textfile collector feature](https://github.com/prometheus/node_exporter#textfile-collector) of the Prometheus node exporter to export metrics to Prometheus that M3 then ingests. To follow the next steps, [download node_exporter](https://github.com/prometheus/node_exporter#installation-and-usage)
+
+#### Configure and Start Prometheus
+
+With M3 running and ready to receive metrics, change your Prometheus configuration to add M3 as `remote_read` and `remote_write` URLs, and as a job. With the configuration below, Prometheus scrapes metrics from two local nodes `localhost:8080` and `localhost:8081` in the `production` group, and one local node `localhost:8082` in the `canary` group:
+
+{{< codeinclude file="docs/includes/prometheus.yml" language="yaml" >}}
+
+Start Prometheus using the new configuration with the command below:
+
+```shell
+prometheus --config.file=prometheus.yml
+```
+
+#### Start Node Exporters
+
+The three commands below simulate three point of sale (POS) devices reporting an hourly sales total for the POS. The node_exporter textfile collector uses[ _.prom_ files](https://prometheus.io/docs/instrumenting/exposition_formats/) for metrics, and only loads files from a directory, so this requires some extra steps for each node for this example.
```

Comment:
I like the idea of using the textfile collector; however, it seems a bit involved. is there a reason why we can't just start up node exporter and query the standard set of metrics that it exposes? Or perhaps cut down the number of nodes that are started?

Reply:
@benraskin92 Two reasons…

I wanted to show how to use "your metrics" as opposed to to just default ones, second I wanted those metrics to emulate those we use in the other quoickstarts for some consistency in examples.